tests/full-disk-encryption.spec.ts

```diff
@@ -14,10 +15,12 @@ test.describe('The main page', () => {
 
     test('Full-disk encryption', async ({ page }) => {
         await test.step("Select the product", async () => {
-            await page.getByLabel('SUSE ALP Server').check();
-            await page.getByRole("button", { name: "Select" }).click();
+		const productSelectionPage = new ProductSelectionPage(page);
+		await productSelectionPage.checkSuseAlpServer();
```

Comment:
We will get the product name from environment variable, refer to ticket https://progress.opensuse.org/issues/133562 and my PR https://github.com/jknphy/e2e-agama-playwright/pull/9

Reply:
If outside just call checkSuseAlpServer, the name change will not impact the test logic.

For inside product page, we have to add extra four envs:
process.env.PRODUCTNAME_ALP=
process.env.PRODUCTNAME_MICRO=
process.env.PRODUCTNAME_TW=
process.env.PRODUCTNAME_LEAP=

What's your suggestions?  @lemon-suse @rakoenig

Reply:
We won't have micro any more, 'SUSE ALP Dolomite' don't have the product selection, only TW and Leap16 has the product selection. Why need so many environment variables? We just need one to set the current product name.